src/app/app.module.js

```diff
@@ -15,8 +15,8 @@
             'myApp.components.notifications',
```

Comment:
Modules are meant to represent a logical grouping to the angular application, where some parts of the application might be self-contained enough to be used individually in other applications. Over here, components are pretty coupled, which means that they could just be used under the same module, which would remove much of the need for having different modules for every component and then spamming them all in this module declaration.